commerce_coordinator/apps/commercetools/tasks.py

```diff
@@ -38,10 +39,11 @@ def update_line_item_state_on_fulfillment_completion(
             from_state_id,
             to_state_key
         )
+        logger.info(f"[update_line_item_state] Successfully updated line item {line_item_id} for order {order_id}.")
         return updated_order
     except CommercetoolsError as err:
-        logger.error(f"Unable to update line item [ {line_item_id} ] state on fulfillment "
-                     f"result with error {err.errors} and correlation id {err.correlation_id}")
+        logger.error(f"[update_line_item_state] Failed to update line item {line_item_id} for order {order_id}. "
+                     f"Error: {err.errors}, Correlation ID: {err.correlation_id}")
```

Comment:
Can we make this a continuous line instead.
```suggestion
                     f"with error: {err.errors}, Correlation ID: {err.correlation_id}")
```

Reply:
Please use continuous lines for logs elsewhere in the PR too

Reply:
This still needs resolution elsewhere. Please use the same approach in the whole PR.